pkg/crypto/clef.go

```diff
@@ -0,0 +1,91 @@
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"errors"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/ethereum/go-ethereum/accounts"
+)
+
+var (
+	ErrNoAccounts       = errors.New("no accounts found in clef")
+	ClefRecoveryMessage = []byte("public key recovery message")
```

Comment:
This variable does not have to be exported, it is used only in tests, so it can be exported with export_test.go only.

Reply:
fixed